near/nep141-locker/src/lib.rs

```diff
@@ -625,32 +557,21 @@ impl Contract {
         let fee = message.amount.0 - fin_transfer.amount.0;
 
         if message.fee.native_fee.0 != 0 {
-            let native_fee_recipient = native_fee_recipient.sdk_expect("ERR_FEE_RECIPIENT_NOT_SET");
-            require!(
-                message.get_origin_chain() == native_fee_recipient.get_chain(),
-                "ERR_WRONG_FEE_RECIPIENT_CHAIN"
-            );
-
             if message.get_origin_chain() == ChainKind::Near {
-                let OmniAddress::Near(near_recipient) = &native_fee_recipient else {
-                    env::panic_str("ERR_WRONG_CHAIN_KIND")
-                };
-                Promise::new(near_recipient.clone())
+                Promise::new(fin_transfer.fee_recipient.clone())
                     .transfer(NearToken::from_yoctonear(message.fee.native_fee.0));
             } else {
-                let required_balance = self.update_fin_transfer(
-                    &message.get_transfer_id(),
-                    &Some(NativeFee {
-                        amount: message.fee.native_fee,
-                        recipient: native_fee_recipient.clone(),
-                    }),
-                );
+                let native_token_address = OmniAddress::new_zero(message.get_origin_chain())
```

Comment:
new_zero -> get_native_token_address. 

It might not be zero if we use wrapped tokens.